metabrainz/__init__.py

```diff
@@ -56,23 +55,23 @@ def create_app(config_path=None):
 
         app.config.from_pyfile(consul_config, silent=True)
 
+    # Printing out some debug values such as config and git commit
+    try:
+        with open(".git-version") as f:
+            git_version = f.read()
+        print('Running on git commit %s' % git_version.strip())
+    except subprocess.CalledProcessError as e:
```

Comment:
This error will probably not be thrown now.

Reply:
Oh, good point.